benchmarks/kernels/benchmark_moe.py

```diff
@@ -585,7 +585,10 @@ def main(args: argparse.Namespace):
         "DeepseekV3ForCausalLM",
         "DeepseekV2ForCausalLM",
         "Glm4MoeForCausalLM",
+        "Glm4vMoeForConditionalGeneration",
     ):
+        if config.architectures[0] == "Glm4vMoeForConditionalGeneration":
```

Comment:
```suggestion
        config = config.get_text_config()
```

HF transformers now supports this API which works for both text-only and multi-modal models